DOCKER.md

```diff
@@ -0,0 +1,82 @@
+# Docker
+
+
+## Prerequisites
+
+The only prerequisites are:
+1. This repo, and
+2. Docker
+
+Building the docker image pulls in all the dev dependencies to build loop within the image itself. Having a `go` development environment is not required.
+
+
+## Building the Docker Image
+
+The docker image can be built using this command within the `loop` directory:
+```
+docker build --tag loop .
+```
+This command pulls down a `go` build container, builds `loop` and `loopd` executables, then publishes those binaries to a fresh, smaller image, and marks that image with the tag 'loop'.
+
+
+## Running the Docker Image
+
+The docker image contains:
+* The binary `loopd`, at `/go/bin/loopd`
+* The binary `loop`, at `/go/bin/loop`
+
+Docker is very flexible so you can use that information however you choose. This guide isn't meant to be prescriptive.
+
+
+### Example: Running loopd
+
+One way of running `loopd` is 
+```
+docker run --rm -it --name loopd -v $HOME/.lnd:/root/.lnd loop:latest loopd --network=testnet --lnd.host <my-lnd-ip-address>:10009
```

Comment:
Probably worth mapping the `~/.loopd` directory as well since it stores the state of active loops.

Reply:
Excellent point! Thanks for that. I've updated that line to map ~/.loop, as well as added a note about why.